telegram/ext/_jobqueue.py

```diff
@@ -821,8 +842,25 @@ def next_t(self) -> Optional[datetime.datetime]:
         return self.job.next_run_time
 
     @classmethod
-    def _from_aps_job(cls, job: "APSJob") -> "Job[CCT]":
-        return job.func.__self__
+    def from_aps_job(cls, job: "APSJob") -> "Job[CCT]":
+        """Provides the :class:`telegram.ext.Job` that is associated with the given APScheduler
+        job.
+
+        Tip:
+            This method can be useful when using advanced APScheduler features along with
```

Comment:
maybe add an `Example:` block here? I think rescheduling a job is fairly common no?

Reply:
rescheduling doesn't require the method though … just `ext_job(.job).modify(…)`

Reply:
oh right, this returns a telegram.ext Job. Okay then nothing more is needed